core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -1585,7 +1591,7 @@ class KafkaZkClient private (zooKeeperClient: ZooKeeperClient, isSecure: Boolean
 
   private def acls(path: String): Seq[ACL] = ZkData.defaultAcls(isSecure, path)
 
-  private def retryRequestUntilConnected[Req <: AsyncRequest](request: Req): Req#Response = {
+  private[zk] def retryRequestUntilConnected[Req <: AsyncRequest](request: Req): Req#Response = {
```

Comment:
Do we need to expose package level visibility for this method?

Reply:
Is to take advantage of that method to perform the call to Zookeeper.

I can move the method `getPathCzxid` present in the class `ExpiredKafkaZkClient` from the test to the class `KafkaZkClient` (but not useful in that class `KafkaZkClient`).

Or I can add more code just to make the call to Zookeeper to get the `czxid` in the class `ExpiredKafkaZkClient`

What do you think?